gdal/doc/source/development/rfc/rfc80_numfocus_relationship.rst

```diff
@@ -0,0 +1,140 @@
+Status:        Development
+============== =============================================
+
+Summary
+-------
+
+The GDAL project will apply for Fiscal sponsorship to `NumFOCUS <https://numfocus.org>`__,
+under the "Grantor-Grantee" model
+
+Motivation
+----------
+
+GDAL's funding model to date has been using the overhead of new feature
+implementation to support the maintenance activities. While this is a common
+open source funding model, it is no longer sustainable given the maturity that
+GDAL has reached. Undirected financial resources will give the project the
+ability to target challenges that have proven too big or too structural to
+otherwise attack.
+
+Details
+-------
+
+The GDAL project has determined that joining NumFOCUS for Fiscal sponsorship
+will help address those challenges. NumFOCUS will receive donations given by
+sponsors of the GDAL project.
+
+GDAL is well aligned with the requirements for a project to join NumFOCUS, most
+of them are inherited from being an OSGeo project:
+
+- Be scientifically oriented.
+- Be open.
+- Be kind.
+
+More precisely, GDAL will apply for Fiscal sponsorship under the "Grantor-Grantee"
+model. It is described at https://numfocus.org/projects-overview and put inline
+below for convenience:
+
+::
+
+    The Grantor-Grantee Model is a "lightweight" option for fiscal sponsorship,
+    for when a project is in need of an entity to receive and manage grant
+    funding. In this model, the project remains a separate entity responsible
+    for managing its own tax and liability issues. NumFOCUS ensures that the
+    project will use the grant funds received to accomplish the ends described
+    in the grant proposal, in alignment with our purpose as a 501(c)(3) public
+    charity in the United States.
+
+
+Funds donated to GDAL will be earmarked to it (minus the commission received by
+NumFOCUS for its operations, typically 15%, to be confirmed).
+
+The project has already submitted a `first document <https://docs.google.com/document/d/1-cZzyctrfvpqF_Cymkn0M9yKYClCA_5MGAmJ_frAi98/edit#heading=h.iq2vc7xizie0>`__ presenting what GDAL is and why it wants to join NumFOCUS:
+
+We now need to formally apply through their `application form <https://numfocus.typeform.com/to/VUPE35>`__
+so that the application is passed to the NumFOCUS board for approval.
+
+The proposed answers to that application form are at:
+https://docs.google.com/document/d/1bc5jdpCe1axdyBHxbJnun7e0DTyDoZI_eFYgJYnOhB8/edit
+
+A `Pre-approved Grantor/ Grantee Agreement <https://docs.google.com/document/d/12KGmSD_EGWqlzh0fiaHNIiN6xPWgdTy--NYf4QR-5k8/edit>`__
+will also have to be signed by 3 members of the GDAL PSC.
+
+Being linked to NumFOCUS, GDAL might be able to benefit from small grants, funded
+by NumFOCUS general budget, and allocated by the NumFOCUS board to projects it will
+select (among the general pool of projects hosted by NumFOCUS). This point will
+be clarified as our relationship with NumFOCUS matures.
```

Comment:
I think we might either omit this section, or clarify that we are primarily expecting funding through NumFOCUS to be from sponsors solicited by the project rather than from NumFOCUS grants from the general budget.

Reply:
Agreed to remove this. This was more for information. But clearly those small grant programs (with unknown outcome) aren't the main motivation to go through NumFOCUS

Reply:
removed